pkg/execution/batch/batch.go

```diff
@@ -78,11 +78,11 @@ type ScheduleBatchOpts struct {
 }
 
 type ScheduleBatchPayload struct {
-	BatchID          ulid.ULID  `json:"batchID"`
-	AccountID        uuid.UUID  `json:"acctID"`
-	WorkspaceID      uuid.UUID  `json:"wsID"`
-	AppID            uuid.UUID  `json:"appID"`
-	FunctionID       uuid.UUID  `json:"fnID"`
-	FunctionVersion  int        `json:"fnV"`
-	FunctionPausedAt *time.Time `json:"fpAt,omitempty"`
+	BatchID                    ulid.ULID  `json:"batchID"`
+	AccountID                  uuid.UUID  `json:"acctID"`
+	WorkspaceID                uuid.UUID  `json:"wsID"`
+	AppID                      uuid.UUID  `json:"appID"`
+	FunctionID                 uuid.UUID  `json:"fnID"`
+	FunctionVersion            int        `json:"fnV"`
+	DeprecatedFunctionPausedAt *time.Time `json:"fpAt,omitempty"` // deprecated
```

Comment:
How long do we need to keep this around? Just until the old consumers are completely phased out and we run an updated version of pkg/execution/executor/service.go:300 that passes FunctionPausedAt to RetrieveAndScheduleBatchWithOpts?

Reply:
Theoretically, I believe we could remove it now, since it's `omitempty` and no code we've shipped in prod has ever set this field.

For safety, I think we should keep this around until all payloads created before this PR is merged have been processed.